pinot-common/src/main/java/org/apache/pinot/common/minion/MinionTaskMetadataUtils.java

```diff
@@ -36,12 +36,25 @@ private MinionTaskMetadataUtils() {
   }
 
   /**
-   * Fetches the ZNRecord for the given minion task and tableName, from MINION_TASK_METADATA/taskName/tableNameWthType
+   * Fetches the ZNRecord for the given minion task and tableName. Fetch from the old path
```

Comment:
I think we should try the new path first, then fall back to the old one if the new one does not exist?

Reply:
okay, updated!